docs/docs/05-migrations/11-v7-to-v8.md

```diff
@@ -135,7 +147,7 @@ govRouter.AddRoute(govtypes.RouterKey, govv1beta1.ProposalHandler).
 - AddRoute(ibcclienttypes.RouterKey, ibcclient.NewClientProposalHandler(app.IBCKeeper.ClientKeeper))
 ```
 
-Remove the UpgradeProposalHandler from the BasicModuleManager
+Remove the `UpgradeProposalHandler` and `UpdateClientProposalHandler` from the `BasicModuleManager`
```

Comment:
This will likely need to be updated:
- remove section on removing proposal handler
- remove section on removeing the update client proposal handler from app module basic

Reply:
I updated this part, but according to the changes:

- Mentioned that both `UpgradeClientProposal` and `UpdateClientProposal` are deprecated.
- Kept the part about removing the update client proposal handler from app module basic, since this can still be removed, right? Isn't this some CLI thing?